server/classes/Lobby/Lobby.ts

```diff
@@ -1,6 +1,5 @@
-import Application from './Application';
-import { Game } from './Game';
-import Player from './Player';
+import Application from '../Application';
+import { Game, Player } from '..';
```

Comment:
From .. ?? What does it do?

Reply:
Yep, sure! So, we're currently in /classes/Lobby/Lobby.ts, right?
If you check in /classes, you'll see there's an index.ts file.

Here's the /classes/index.ts file:
```ts
export { default as Lobby } from './Lobby/Lobby'
export { default as Game } from './Lobby/Game'
export { default as Player } from './Player'
```

This mean that we can, for example, import Lobby from this file:

```ts
import { Lobby } from '/classes/index.ts';
```

But because the file is named index.ts, we don't even have to say the file of the name; TS will know it is the entry point.

So this line:
```ts
import { Game, Player } from '..'; 
```

Could be rewritten into this line:
```ts
import { Game, Player } from '../index';
```

They are exactly the same. 

![image](https://user-images.githubusercontent.com/22148893/140294730-a8db8312-20c1-4f00-9d7a-b6751723dfe3.png)